src/scripts/trim_reblogs.js

```diff
@@ -38,7 +38,9 @@ const onButtonClicked = async function ({ currentTarget: controlButton }) {
       showModal({
         title: 'Note: Legacy post',
         message: [
-          'The root post of this thread was originally created with the legacy post editor.',
+          'This thread was originally created, or at some point edited, using the ',
```

Comment:
lol you were too quick, I think this is more grammatically correct?
```suggestion
          'This thread was originally created, or at some point was edited, using the ',
```

Reply:
wait how did github... what? it looked like that on my screen when I accepted the suggestion but it didn't do it :P